sql-cli/sql_cli/connections.py

```diff
@@ -49,17 +52,19 @@ def validate_connections(connections: list[Connection], connection_id: str | Non
     for connection in connections:
         if not connection_id or connection_id and connection.conn_id == connection_id:
             os.environ[f"AIRFLOW_CONN_{connection.conn_id.upper()}"] = connection.get_uri()
-            status = (
-                "[bold green]PASSED[/bold green]" if _is_valid(connection) else "[bold red]FAILED[/bold red]"
-            )
+            success_status, message = _is_valid(connection)
+            status = "[bold green]PASSED[/bold green]" if success_status else "[bold red]FAILED[/bold red]"
             rprint(f"Validating connection {connection.conn_id:{CONNECTION_ID_OUTPUT_STRING_WIDTH}}", status)
+            formatted_message = (
+                f"[bold green]{message}[/bold green]" if success_status else f"[bold red]{message}[/bold red]"
+            )
+            log.debug("Connection Message: %s", formatted_message)
 
 
-def _is_valid(connection: Connection) -> bool:
+def _is_valid(connection: Connection) -> tuple[bool, str]:
     # Sqlite automatically creates the file if it does not exist,
     # but our users might not expect that. They are referencing a database they expect to exist.
     if connection.conn_type == "sqlite" and not Path(connection.host).is_file():
-        return False
+        return False, "Sqlite db does not exist!"
```

Comment:
Perhaps we could use the opportunity to cover this line with a test!

Reply:
This is being removed in https://github.com/astronomer/astro-sdk/pull/1514.